docs/modules/data-structures/pages/cpmap.adoc

```diff
@@ -41,10 +41,7 @@ that the JVM heap is appropriately sized. There is a maximum `CPMap` size of `20
 
 WARNING: A `CPMap` is a CP data structure and therefore takes part in a process called _snapshotting_.
 When defining the capacity of a `CPMap` you must consider the following memory requirements: 
-
-* on-heap capacity for the `CPMap`; and
-* on-heap capacity for a `CPMap` snapshot for each follower within the respective CP Group. 
-
-Therefore, at a minimum for a `CPMap` with a capacity of `5MB` within a 3-member CP Group,
-we must conservatively ensure that we have on-heap headroom for `15MB`. This capacity planning must
-be considered for all `CPMap` instances you intend to create.
+(1) on-heap capacity for the `CPMap`; and (2) on-heap capacity for a `CPMap` snapshot for each 
+follower within the respective CP Group. Therefore, at a minimum for a `CPMap` with a capacity of 
+`5MB` within a 3-member CP Group, we must conservatively ensure that we have on-heap headroom for 
+`15MB`. This capacity planning must be considered for all `CPMap` instances you intend to create.
```

Comment:
To make it work, format the warning as follows:

[WARNING]
====
A `CPMap` is a CP data....following memory requirements:

* On-heap capacity for the `CMap`
* On-heap capacity for a CMAP snapshot for each following within the respective CP Group

Therefore, the minimum required for a `CPMap` with a capacity of 5MB in a three-member group, must conservatively ensure that....
====

Reply:
Done, that seems to have done the trick!